docker-compose.yml

```diff
@@ -72,6 +72,7 @@ services:
   postgres:
     image: postgis/postgis:13-3.1-alpine
     environment:
+      POSTGRES_USER: directus
```

Comment:
This was needed to get the local API service to connect to the postgres docker container.